website/docs/data-display/bar-horizontal/bar-horizontal.md

```diff
@@ -83,30 +83,32 @@ Basic data visualization rules are described in the [D3 chart](/data-display/d3-
 
 ## Usage
 
-**Use horizontal bars when:**
+**Use Horizontal bar chart when:**
 
-- Categories have long names, like country names.
-- Comparing more than 10 categories.
+- Comparing less than 10 categories.
 - Each category's value needs labeling.
 - Showing category distribution in order.
 
-**Avoid horizontal bars when:**
+**Use [Cigarette pack bar chart](/data-display/cigarette-pack/cigarette-pack) instead of Horizontal bar chart when:**
```

Comment:
They are actually differs a little bit in when they should be used, but overall I agree. I was doubting about adding something more straightforward in the text. Will add. It could be helpful indeed.